documentation/src/main/asciidoc/mapping.asciidoc

```diff
@@ -456,32 +468,43 @@ public class Essay {
     ...
 
     @Id
-    @DocumentId
+    @GeneratedValue
```

Comment:
Feel free to keep this, just a though to share:
I typically assume it's ok in our examples to assume "application assigned ids", so to not have to add annotations which might be distracting / non essential to the example.